tests/blas_like/Axpy.cpp

```diff
@@ -0,0 +1,137 @@
+/*
+   Copyright (c) 2009-2016, Jack Poulson
+   All rights reserved.
+
+   This file is part of Elemental and is under the BSD 2-Clause License,
+   which can be found in the LICENSE file in the root directory, or at
+   http://opensource.org/licenses/BSD-2-Clause
+*/
+#include <El.hpp>
+using namespace El;
+
```

Comment:
I've added a simple correctness test where we compute Axpy on a distributed matrix, then have rank 0 compute the error at each entry. When I run the test, I get no numerical error.